src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -1709,38 +1709,39 @@ private bool SkipComment()
 
         private bool SkipSingleLineComment(ReadOnlySpan<byte> localBuffer, out int idx)
         {
-            idx = localBuffer.IndexOfAny(JsonConstants.LineFeed, JsonConstants.CarriageReturn);
-            if (idx == -1)
+            idx = localBuffer.IndexOfAny(JsonConstants.CarriageReturn, JsonConstants.LineFeed);
```

Comment:
I am curious, why invert these? I would imagine `LineFeed` to be more common, especially on Unix.

Reply:
Inverted this with view of matching how we would typically read CR-LF. Didn't think about prevalent and frequent use of LF, to make it the first argument. Reverting.